colmena/redis/queue.py

```diff
@@ -26,9 +26,8 @@ def make_queue_pairs(hostname: str, port: int = 6379, name='method',
                      keep_inputs: bool = True,
                      clean_slate: bool = True,
                      topics: Optional[Iterable[str]] = None,
-                     value_server_threshold: Optional[int] = None,
-                     value_server_hostname: Optional[str] = None,
-                     value_server_port: Optional[int] = None)\
+                     proxystore_name: Optional[str] = None,
```

Comment:
Could change to `proxystore_names: Optional[Union[str, Iterable[str]]] = None` to support different backends per topic or one backend for all topics`.

Reply:
After thinking about this more, I think it would be good to allow the user to associate different ProxyStores with different topics. The only weird thing is handling the 'default' topics, because it's not necessarily apparent to the user that the 'default' topic will be appended to their topics list. 

For example, the user passes `topics=['simulate', 'infer', 'train']` and `proxystore_name=['redis', 'globus', 'globus']`. What ProxyStore should we use for the 'default' queue?

We could instead say if the users passes `proxystore_name`, it should be a dictionary containing a mapping of topics to ProxyStores and say in the docstring that the 'default' topic should be included in this mapping (or maybe raise a warning if it is not in the mapping)?

Reply:
I'm totally on board with that plan.